cuBERT_topic_modelling/tests/test_ctfidf.py

```diff
@@ -0,0 +1,95 @@
+from bertopic import BERTopic
+import pandas as pd
+import numpy as np
+import cudf
+from sklearn.datasets import fetch_20newsgroups
+import pytest
+from cupyx.scipy.sparse.csr import csr_matrix
+from cuBERTopic import gpu_BERTopic
+from ctfidf import ClassTFIDF
+from vectorizer.vectorizer import CountVecWrapper
+
+data_trivial = [
+    "This is the first document.",
+    "This document is the second document.",
+    "And this is the third one.",
+    "Is this the first document?",
+]
+
+newsgroup_docs = fetch_20newsgroups(
+    subset="all", remove=("headers", "footers", "quotes")
+)["data"][:1000]
+
+docs_df_trivial = pd.DataFrame(data_trivial, columns=["Document"])
+docs_df_trivial["Topic"] = [1, 2, 0, 1]
+docs_df_trivial = docs_df_trivial.sort_values("Topic")
+
+data_big = fetch_20newsgroups(subset="all")["data"]
+docs_df_big = pd.DataFrame(data_big, columns=["Document"])
+docs_df_big["Topic"] = np.random.randint(0, 100, len(docs_df_big))
+docs_df_big = docs_df_big.sort_values("Topic")
+
```

Comment:
Look into pytest fixtures. Global variables should be avoided as much as possible.

Reply:
I think the `request.getfixturevalue` is a better solution. We should use that. 


Related, Can you create a `conda-env` file for the dev environment to ensure we can actually run this example.

Reply:
based on the issue I linked here: https://github.com/pytest-dev/pytest/issues/4666, it doesn't seem feasible to use it with parametrized inputs (also mentioned at https://stackoverflow.com/questions/42014484/pytest-using-fixtures-as-arguments-in-parametrize). Thoughts?

Reply:
In that case feel free to use the pytest-lazy-fixture . This is anyways a dev only dependency so its fine.